site/content/en/docs/talks_and_presentations/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Talks and presentations"
+linkTitle: "Talks and presentations"
+weight: 40
+menu:
+  main:
+    weight: 40
+description: >
+  Talks and presentations showcasing Kueue
+---
+
+# 2024
+
+* KubeCon, Nov'24, Salt Lake City
+  [Keynote: Multicluster Batch Jobs Dispatching with Kueue at CERN](https://youtu.be/xMmskWIlktA)
+  Ricardo Rocha, Lead Platforms Infrastructure, CERN & Marcin Wielgus, Staff Software Engineer, Google
```

Comment:
```suggestion
  Ricardo Rocha, CERN & Marcin Wielgus, Google
```

Are there any specific motivations the reason why we want to mention Job title only here?